doc/start/hardware-recommendations.rst

```diff
@@ -156,27 +156,29 @@ fully but prudently.
 Data Storage
 ============
 
-Plan your data storage configuration carefully. There are significant cost and
-performance tradeoffs to consider when planning for data storage. Simultaneous
+Plan your data storage configuration carefully: there are significant cost and
+performance tradeoffs to consider. Routine
 OS operations and simultaneous requests from multiple daemons for read and
-write operations against a single drive can impact performance.
+write operations against a single drive can impact performance and stability.
 
 OSDs require substantial storage drive space for RADOS data. We recommend a
-minimum drive size of 1 terabyte. OSD drives much smaller than one terabyte
+minimum OSD size of 1 tebibyte (TiB). OSD drives much smaller than this
 use a significant fraction of their capacity for metadata, and drives smaller
 than 100 gigabytes will not be effective at all.
```

Comment:
Should this be also modified to GiB to be consistent with your [previous change on line 165](https://github.com/ceph/ceph/pull/66293/files#diff-d1c8bb767c69fecb9beeef3856f535d094658b61cc8ca66bfa567afb582cfebcR165)?